### DISCUSSION TITLE
Move a modified file does not display changes

### DISCUSSION THREAD

**Discussion by ruudk** [rectorphp/rector#6952] [Q&A]
I'm using:
```php
        $this->removedAndAddedFilesCollector->addMovedFile($this->file, $newFileLocation);
```
to move a modified node to a new location.

But the output only shows:
```
...
/opt/homebrew/opt/php@8.0/bin/php -dxdebug.mode=debug /Volumes/CS/www/website/vendor/bin/rector process --debug --xdebug --dry-run src/Core/ResaleTicketingBundle/Repository

// ....  all rectors that were executed 

 ! [NOTE] File                                                                  
 !        "src/Core/ResaleTicketingBundle/Repository/TransactionRepository.php" 
 !        will be moved to                                                      
 !        "/Volumes/CS/www/website/src/Core/ResaleTicketingBundle/Repository/Doc
 !        trineTransactionRepository.php"                                       

 [OK] Rector is done!                                                           
```

When I don't move the file, it displays the diff.

Is there a reason why the diff is not shown?

**Comment by TomasVotruba** [comment]
Hi, this is on purpose as moved files were displayed as full content with sometimes hundreds of lines.
For these cases, it's better to review the result in pull-request on Github/Gitlab, as Rector diffs are only informatory not to replace review tools.

**Reply by ruudk** [reply]
But in case of modified node, couldn't it just diff it with the old class node? And show only the diff? If nothing was changed, then no diff (to prevent large diff)

**Reply by TomasVotruba** [reply]
We had such design before, but the output was confusing as the original file did not change. It was moved.

**Reply by ruudk** [reply]
When file is moved and the node did not change > show "File moved from A to B"
When file is moved and the node changed too > show "File moved from A to B" + show the diff (it should only show the tiny changes that where done)

How can that be confusing?

**Reply by TomasVotruba** [reply]
The diff is connected with file content change. While no diff and message is connected with file move.
